apps/librelingo_json_export/tests/test_export_is_dry_run.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from librelingo_json_export.export import _is_dry_run
+from librelingo_json_export.cli import Settings, DEFAULT_SETTINGS
+
+
+def test_returns_true_when_dry_run():
+    assert _is_dry_run(Settings(True))
```

Comment:
ok, so in the end if we do get rid of this function (unless it's really needed for something else) then these tests won't be needed either

Reply:
regarding this function I'm actually wondering why it's needed. How come `settings` can be `None`? I think it should always be supplied, no? 

I'm concerned that if we follow this logic, then a lot of similar fallback functions could be created if more settings value are used in a similar way where `settings` could be `None`.

If it's absolutely impossible to avoid having `settings` as `None`, then to prevent the proliferation of such functions that provide a fallback value for each settings value separately, why not create a logic that creates a fallback settings value?

```python
def get_settings(settings):
  if not settings:
    return DEFAULT_SETTINGS

  return settings
```

Reply:
Since `collections.namedtuple` is immutable, I decided to simply use `DEFAULT_SETTINGS` as a default arguments of `settings` in functions: `_export_course_skills`, `_export_skill`, `_export_course_data` and `export_course`. Before it was `None`. Such change does not change the _interface_ too much.

Reply:
amazing, that's the best solution